src/registrar/tests/test_models_domain.py

```diff
@@ -549,6 +550,19 @@ def tearDown(self):
         PublicContact.objects.all().delete()
         Domain.objects.all().delete()
 
+    def run_disclose_security_emails(self):
+        """
+        This method executes the disclose_security_emails command.
```

Comment:
**blocking** this doesn't call your command and shouldn't go here. You should have another test for testing your disclose command works properly

Reply:
🤦‍♀️ just realized that this script runs extend_expiration_dates instead of disclose_emails - thank you for catching that. With that update, should this test still be somewhere else?

Reply:
Hey this part about moving it still needs to be addressed, please move this into the approriate class test_management_scripts.py